### PR TITLE
Re-adds unit tests

### PR DESCRIPTION
**Pull request by ike709**
I commented out all of the broken ones. Getting the test framework back in is more important that fixing every test, we can address those later.

### CODE REVIEW

File: Content.Tests/SetupCompileDM.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
+using System;
+using System.IO;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+
+// This it outside of any namespace so it affects the whole assembly.
```
Comment (wixoaGit) [inline Content.Tests/SetupCompileDM.cs:6]:
```suggestion
// This is outside of any namespace so it affects the whole assembly.
```

File: OpenDreamRuntime/EntryPoint.cs [hunk 2]
```diff
@@ -40,7 +40,7 @@ public class EntryPoint : GameServer {
 
         public override void PostInit() {
             _commandSystem = EntitySystem.Get<DreamCommandSystem>();
-            _dreamManager.Initialize();
+            _dreamManager.Initialize(null);
```
Comment (wixoaGit) [inline OpenDreamRuntime/EntryPoint.cs:43]: Have this call pass the cvar instead of having multiple locations default to `testingJson ?? _configManager.GetCVar<string>(OpenDreamCVars.JsonPath)`

**Review by wixoaGit — CHANGES_REQUESTED** [review]

File: OpenDreamRuntime/DreamManager.cs [hunk 3]
```diff
@@ -39,16 +39,17 @@ partial class DreamManager : IDreamManager {
         public List<DreamObject> Datums { get; set; } = new();
         public Random Random { get; set; } = new();
 
-        public void Initialize() {
+        //TODO This arg is awful and temporary until RT supports cvar overrides in unit tests
+        public void Initialize(string? testingJson) {
```
Comment (wixoaGit) [inline OpenDreamRuntime/DreamManager.cs:43]: This is more than just the testing json now, and shouldn't be null.
```suggestion
        public void Initialize(string jsonPath) {
```

File: OpenDreamRuntime/DreamManager.cs [hunk 4]
```diff
@@ -39,16 +39,17 @@ partial class DreamManager : IDreamManager {
...
             InitializeConnectionManager();
 
-            DreamCompiledJson json = LoadJson();
+            DreamCompiledJson json = LoadJson(testingJson);
```
Comment (wixoaGit) [inline OpenDreamRuntime/DreamManager.cs:46]:
```suggestion
            DreamCompiledJson json = LoadJson(jsonPath);
```

File: OpenDreamRuntime/DreamManager.cs [hunk 5]
```diff
@@ -39,16 +39,17 @@ partial class DreamManager : IDreamManager {
...
             if (json == null)
                 return;
 
             _compiledJson = json;
 
-            _dreamResourceManager.Initialize();
+            _dreamResourceManager.Initialize(testingJson);
```
Comment (wixoaGit) [inline OpenDreamRuntime/DreamManager.cs:52]:
```suggestion
            _dreamResourceManager.Initialize(jsonPath);
```

File: OpenDreamRuntime/DreamManager.cs [hunk 6]
```diff
@@ -83,7 +84,11 @@ partial class DreamManager : IDreamManager {
                 globalInitProc.Spawn(WorldInstance, new DreamProcArguments(new(), new()));
             }
 
-            _dreamMapManager.LoadMaps(json.Maps);
+            //TODO figure out why this json doesn't work
+            if (testingJson is null)
+            {
+                _dreamMapManager.LoadMaps(json.Maps);
+            }
```
Comment (wixoaGit) [inline OpenDreamRuntime/DreamManager.cs:91]: Is this still an issue?

**Review by wixoaGit — CHANGES_REQUESTED** [review]

File: OpenDreamRuntime/DreamManager.cs (hunk 6 above)
Comment (ike709) [reply OpenDreamRuntime/DreamManager.cs]: No

**Review by wixoaGit — APPROVED** [review]
